MoEngagePluginCards.podspec

```diff
@@ -0,0 +1,33 @@
+Pod::Spec.new do |s|
+    s.name         = 'MoEngagePluginCards'
+    s.version      = '1.0.0'
+    s.summary      = 'MoEngage Cards Plugin for Hybrid SDKs'
+    s.description  = <<-DESC
+    MoEngage is a mobile marketing automation company. This framework is used by our plugins built for different hybrid frameworks i.e, Flutter, Cordova, React Native etc.
+                     DESC
+
+    s.homepage     = 'https://www.moengage.com'
+    s.documentation_url = 'https://developers.moengage.com/'
+    s.license      = { :type => 'Commercial', :file => 'LICENSE' }
+    s.author       = { 'MobileDevs' => 'mobiledevs@moengage.com' }
+    s.social_media_url   = 'https://twitter.com/moengage'
+    s.platform     = :ios
+    s.ios.deployment_target = '11.0'
+    
+    s.source       = {
+                        :git => 'https://github.com/moengage/iOS-PluginBase.git',
+                        :tag => 'pluginCards-' + s.version.to_s
+                    }
+
+    s.source_files = 'MoEngagePluginCards/**/*'
+    s.swift_version = '5.0'
+    s.frameworks = 'UIKit', 'Foundation', 'UserNotifications'
+    s.dependency 'MoEngagePluginBase', '>= 4.2.0', '< 4.3.0'
+    # TODO: update constraints after release
+    s.dependency 'MoEngageCards', '>= 4.8.0', '< 4.9.0'
+
+    s.test_spec 'UnitTests' do |ts|
+      ts.source_files = 'Tests/MoEngagePluginCardsTests/**/*.swift'
```

Comment:
@msoumya-engg-sdk lets not include unit test cases as part of library when someone is installing

Reply:
@RakshithaAcharya  user has to specifically request for test specs to be installed, by default test specs aren't installed as per [docs](https://guides.cocoapods.org/syntax/podfile.html#pod): 
> Test specs can be optionally included via the `:testspecs` option. By default, none of a Pod's test specs are included.

This gives the advantage of running the tests when running `pod lib lint` command. What are your thoughts?
cc. @umangmoe @babulmoengage

Reply:
@msoumya-engg-sdk then its fine.